cinema/models.py

```diff
@@ -84,24 +84,32 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
+    @staticmethod
+    def validate_ticket(row, seat, cinema_hall, error_to_raise):
         for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
+            (row, "row", "rows"),
+            (seat, "seat", "seats_in_row"),
         ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+            count_attrs = getattr(cinema_hall, cinema_hall_attr_name)
             if not (1 <= ticket_attr_value <= count_attrs):
-                raise ValidationError(
+                raise error_to_raise(
                     {
-                        ticket_attr_name: f"{ticket_attr_name} "
+                        ticket_attr_name:
+                        f"{ticket_attr_name} number "
                         f"number must be in available range: "
                         f"(1, {cinema_hall_attr_name}): "
                         f"(1, {count_attrs})"
```

Comment:
The error message formatting in the `validate_ticket` method is incorrect. The message should be formatted as a single string. Consider using parentheses to concatenate the strings properly, or use an f-string for better readability and correctness.